Scripts/modelsystem.py

```diff
@@ -165,7 +165,6 @@ def _add_internal_demand(self, previous_iter_impedance, is_last_iteration):
                 if purpose.dest != "source":
                     for mode in demand:
                         self.dtm.add_demand(demand[mode])
-                    log.info(f"Time period matrices calculated for {purpose.name}")
```

Comment:
This is part of the second heavy loop of the demand model. But the most time-consuming step of that loop, `calc_demand` is already logged, so this does not add much useful information.